nservicebus/upgrades/asp-6to1.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Azure Storage Persistence Upgrade Version 6 to 1
```

Comment:
The problem with the naming is not just clarity but length. My understanding is that the `title` header info is used to render the name in the menu. The longer the name the weirder it's going to look in the doco menu. 

The menu bar is very narrow which means that the long suggestion will word wrap multiple times. There is precedence for  this (see Azure BlobStorage Databus below) but I'm not sure how doco maintainers feel about this stylistically.

![image](https://cloud.githubusercontent.com/assets/1378057/15297739/ecf488e4-1b58-11e6-85a4-ffb3850796c9.png)